protoc-gen-grpc-gateway/gengateway/template.go

```diff
@@ -23,6 +23,14 @@ type binding struct {
 	*descriptor.Binding
 }
 
+// GetBinding returns the binding body
+func (b binding) GetBinding() string {
+	if b.Body.FieldPath.String() != "" {
```

Comment:
How certain are you that `Body` and `FieldPath` are never `nil`? I'm happy with leaving it like this so long as you're certain you won't cause a `panic` from it, but if so please leave a comment ensuring that these will never be `nil`.

Otherwise, you should nil check each property.

Reply:
I have updated the PR to check nil properly.